api.web.yaml

```diff
@@ -150,6 +150,32 @@ paths:
         default:
           description: Unexpected error
 
+  /people/{id}:
+    post:
+      parameters:
+        - in: path
+          name: id
+          required: true
```

Comment:
Would it be worth adding a description here to tell us what the `id` represents?